Utilities/PrimeTests.cs

```diff
@@ -0,0 +1,170 @@
+using System.Collections.Generic;
+
+namespace PrimeCalculationsTests
+{
+    [TestClass]
+    public class PrimeTests
+    {
+        //--------------------------------------------------------------------
+        // IsPrime
+        //--------------------------------------------------------------------
+        [TestMethod]
+        public void IsPrime_Test1()
+        {
+            Assert.AreEqual(true, PrimeCalc.IsPrime(2));
+        }
+
+        [TestMethod]
+        public void IsPrime_Test2()
+        {
+            Assert.AreEqual(true, PrimeCalc.IsPrime(13));
+        }
+
+        [TestMethod]
+        public void IsPrime_Test3()
+        {
+            Assert.AreEqual(true, PrimeCalc.IsPrime(1097));
+        }
+
+        [TestMethod]
+        public void IsPrime_Test4()
+        {
+            Assert.AreEqual(true, PrimeCalc.IsPrime(7193));
+        }
+
+        //--------------------------------------------------------------------
+        // GetPrimeNumber
+        //--------------------------------------------------------------------
+        [TestMethod]
+        public void GetPrimeNumber_Test1()
+        {
+            Assert.AreEqual(5, PrimeCalc.GetPrimeNumber(3));
+        }
+
+        [TestMethod]
+        public void GetPrimeNumber_Test2()
+        {
+            Assert.AreEqual(149, PrimeCalc.GetPrimeNumber(35));
+        }
+
+        [TestMethod]
+        public void GetPrimeNumber_Test3()
+        {
+            Assert.AreEqual(541, PrimeCalc.GetPrimeNumber(100));
+        }
+
+        [TestMethod]
+        public void GetPrimeNumber_Test4()
+        {
+            Assert.AreEqual(7919, PrimeCalc.GetPrimeNumber(1000));
+        }
+
+        //--------------------------------------------------------------------
+        // GetLargestPrime
+        //--------------------------------------------------------------------
+        [TestMethod]
+        public void GetLargestPrime_Test1()
+        {
+            Assert.AreEqual(7, PrimeCalc.GetLargestPrime(10));
+        }
+
+        [TestMethod]
+        public void GetLargestPrime_Test2()
+        {
+            Assert.AreEqual(97, PrimeCalc.GetLargestPrime(100));
+        }
+
+        [TestMethod]
+        public void GetLargestPrime_Test3()
+        {
+            Assert.AreEqual(691, PrimeCalc.GetLargestPrime(700));
+        }
+
+        [TestMethod]
+        public void GetLargestPrime_Test4()
+        {
+            Assert.AreEqual(7993, PrimeCalc.GetLargestPrime(8000));
+        }
+
+
+        //--------------------------------------------------------------------
+        // GetListOfPrimes
+        //--------------------------------------------------------------------
+        [TestMethod]
+        public void GetListOfPrimes_Test1()
+        {
+            List<int> list = new List<int>() { 2, 3, 5 };
+
+            CollectionAssert.AreEqual(list, PrimeCalc.GetListOfPrimes(3));
+            //CollectionAssert.AreEquivalent(list, PrimeCalc.GetListOfPrimes(3));

```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6357239)